.circleci/config.yml

```
@@ -170,19 +170,30 @@ jobs:
       - rust_components
       - cache_restore
       - run:
-          name: Check that the workspace hack crate contains all features in use
+          name: Configure git
           command: |
-            cargo hakari generate --diff || {
-              echo "If this fails, fix it by running \`cargo hakari generate\` locally and committing the changes"
-              exit 1
-            }
+            git config user.email "49699333+dependabot[bot]@users.noreply.github.com"
+            git config user.name "dependabot[bot]"
```

Since this will push to ALL PRs, not just the dependabot ones, I think we should use a neutral config here. How about:

```suggestion
            git config user.email "????"
            git config user.name "CircleCI[bot]"
```

(I'm not sure about the email address though)

---

Fun fact: GitHub doesn't care at all about whether this email address is valid or actually belongs to the user making the commit ;) So i've set it to circleci@influxdata.com, which doesn't need to actually exist? Does that seem ok?